library/globals.inc.php

```diff
@@ -695,6 +695,13 @@ function gblTimeZones()
             xl('Option to support inventory and sales of products')
         ),
 
+        'default_visit_category' => [
+            xl('Default Visit Category'),
+            'visit_category',
```

Comment:
Would make this more specific since there are several hundred of these, and we want a search of them to be specific (for example, if search for visit_category, would also get line in edit_globals. Maybe something like default_visit_category